docs/orchestration/global-hooks/overview.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Global Hooks
+sidebarTitle: "Overview"
+description: "Global Hooks are a powerful feature in Mage that allows you to execute custom functions before or after an API request. These hooks can be used to perform additional operations, validate data, or integrate with external systems."
+---
+<img
+  alt="Hook"
+  src="https://github.com/mage-ai/assets/blob/main/Hook.gif?raw=True"
+/>
+
+### **How Global Hooks Work**
+
+Global Hooks can be triggered at two different points during the execution of a pipeline:
+
+1. **Pre-completion of a Block**: The hook will run before a specific block in the pipeline is executed.
+2. **Post-completion of a Block**: The hook will run after a specific block in the pipeline has completed execution.
+
+### **Configuring Global Hooks**
+
+The Global Hooks UI in Mage provides several configuration options to customize the behavior of your hooks. Here's a breakdown of the available settings:
+
+**Hook UUID:** A unique identifier for the hook. This is used to reference the hook within your pipelines.
+
+**Resource Type:** The type of API resource that the hook is attached to (e.g., Pipeline, Dataset, etc.).
+
+**Operation Type:** The operation that triggers the hook (e.g., Create, Update, Delete).
+
+### **Targeting Conditions**
+
+The Targeting Condition fields in the Global Hooks UI allows you to specify a set of conditions that determine which subset of pipelines the hook should run on. This feature gives you control over the hook’s execution of a subset of features.
+
+### **When to Run a Hook**
+
+**Before/After Operation Toggles:** Toggles to specify whether the hook should run before or after the operation.
+
+**Run if Operation Succeeds/Fails Toggles:** Toggles to specify whether the hook should run only if the operation succeeds or fails.
+
+### Code to Run
+
+When a hook is triggered it will execute the pipeline that as selected in the pipeline to execute dropdown.
```

Comment:
nit: comma after "When a hook is triggered"

Reply:
Added comma, tested in mint dev.